scripts/start-full-kessel.sh

```
@@ -6,11 +6,16 @@ source ./scripts/check_docker_podman.sh
 COMPOSE_DIR="development/full-kessel"
 ENV_FILE="${COMPOSE_DIR}/.env"
 
-# Load .env for schema handling
+# Load .env defaults without overriding caller's environment
 if [ -f "${ENV_FILE}" ]; then
+  _saved_schema_zed_file="${SCHEMA_ZED_FILE:-}"
+  _saved_rbac_config_file="${RBAC_CONFIG_FILE:-}"
   set -a
   source "${ENV_FILE}"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)